### PR TITLE
Update Packages

### PR DESCRIPTION
**Pull request by naveen521kk**

### CODE REVIEW

**Comment by MehdiChinoune** [comment]
numba doesn't support (work with) numpy>1.22.

**Comment by naveen521kk** [comment]
I'll drop that commit then, thanks.

File: mingw-w64-python-ipython/PKGBUILD [hunk 1]
```diff
@@ -24,6 +24,7 @@ depends=("${MINGW_PACKAGE_PREFIX}-python-backcall"
          "${MINGW_PACKAGE_PREFIX}-python-prompt_toolkit"
          "${MINGW_PACKAGE_PREFIX}-python-pygments"
          "${MINGW_PACKAGE_PREFIX}-python-setuptools"
+         "${MINGW_PACKAGE_PREFIX}-python-stack-data"
```
Comment (MehdiChinoune) [inline mingw-w64-python-ipython/PKGBUILD:27]: There is no such package!

Comment (naveen521kk) [reply mingw-w64-python-ipython/PKGBUILD:27]: I noticed that I should create one.

**Pull request closed without merging**